src/sections/Landscape/LandscapeGrid.style.js

```diff
@@ -10,6 +10,10 @@ export const LandscapePageWrapper=styled.div`
 		font-style:unset;
 		color:#1e2117;
 	}
+	.sub-heading {
```

Comment:
Doing this, makes the other headers in the page overlap with the above text. Please update.